docs/docs/guides/architecture-scenarios/b2b.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: B2B
+---
+
+import { B2B } from '../../../src/components/b2b';
+
+## Business to Business
+
+B2B describes the situation where an organization interacts with other organizations.
+This **multiple organization architecture** usually adds some form of complexity to an Identity and Access Management System.
+In ZITADEL a B2B organization represents a business partner or partner who typically has its own branding and has different access settings like an additional federated login for its users.
+
+B2b can be a simple scenario where an organization only shares one of its projects with another organization or have a more complex case where an organization is offering a portal application to all its partners with included (self)administration.
+
+<!-- This guide describes an application  -->
+## Sample scenario
+
+Octagon is a fictitious company which is used throughout this guide to explain the details and key concepts of such a B2B scenario.
+Octagon tries to solve multiple tasks in the banking field. Its portfolio includes several applications for their employees, customers, and partners. Some of which are web-based, some of which are used by machine users only.
+
+### Portal Application
+
+Octagon has a **Portal application** where its employees can access their account and list all applications they are allowed to use.
+Employees work for a department within Octagon or for Octagon itself. 
+Some of the users have enhanced features because they supervize certain teams. Those can onboard new employees and manage their roles and features.
```

Comment:
```suggestion
Some of the users have enhanced features because they supervise certain teams. Those can onboard new employees and manage their roles and features.
```